### ISSUE TITLE
Doxywizard condensed configuration view missing IMAGE_PATH

### ISSUE THREAD

**Issue by dhebbeker** [doxygen/doxygen#8372]
**Describe the bug**
In doxywizard, the condensed configuration view is missing the `IMAGE_PATH` entry.

**Expected behavior**
Expected same output as `doxygen -x`:

```
# Difference with default Doxyfile 1.9.1 (ef9b20ac7f8a8621fcfc299f8bd0b80422390f4b)
IMAGE_PATH             = img
```

**Screenshots**
Actual output:

```
# Doxyfile 1.9.1
```

**To Reproduce**

A Doxyfile consisting of this is sufficient:
```
IMAGE_PATH             = img
```

1. Open configuration with Doxywizard
2. Check "Condensed" option in Doxywizard in the "Run" tab
3. Click "Show configuration"

**Version**

 - Windows 10, 64 bit
 - Doxygen version: 1.9.1 (ef9b20ac7f8a8621fcfc299f8bd0b80422390f4b)
 - using official binary from website 
 - Doxywizard version: 1.9.1 (ef9b20ac7f8a8621fcfc299f8bd0b80422390f4b), Qt version: 4.8.7

**Comment by albert-github** [comment]
This is indeed a problem in 1.9.1, a proposed pull request #8346 was already created.
This problem only focused at the `red` color in the expert tab but the condensed output also goes through the the same method `isDefault` and suffers from the same problem.

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.2.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
